### ISSUE TITLE
Canot untar the repo -- "tar xvf" fails with the error "Cannot create symlink ....  File name too long"

### ISSUE THREAD

**Issue by khronosschoty** [MoonchildProductions/moebius#265]
The error below is what takes place when I try to untar the repo. 

Removing the offending README fixes the issue. 

> tar: moebius-2017.12.25media/libav/README: Cannot create symlink to ‘Libav\n=====\n\nLibav is a collection of libraries and tools to process multimedia content\nsuch as audio, video, subtitles and related metadata.\n\n## Libraries\n\n* `libavcodec` provides implementation of a wider range of codecs.\n* `libavformat` implements streaming protocols, container formats and basic I/O access.\n* `libavutil` includes hashers, decompressors and miscellaneous utility functions.\n* `libavfilter` provides a mean to alter decoded Audio and Video through chain of filters.\n* `libavdevice` provides an abstraction to access capture and playback devices.\n* `libavresample` implements audio mixing and resampling routines.\n* `libswscale` implements color conversion and scaling routines.\n\n## Tools\n\n* [avconv](http://libav.org/avconv.html) is a command line toolbox to\n  manipulate, convert and stream multimedia content.\n* [avplay](http://libav.org/avplay.html) is a minimalistic multimedia player.\n* [avprobe](http://libav.org/avprobe.html) is a simple analisys tool to inspect\n  multimedia content.\n* Additional small tools such as `aviocat`, `ismindex` and `qt-faststart`.\n\n## Documentation\n\nThe offline documentation is available in the **doc/** directory.\n\nThe online documentation is available in the main [website](http://libav.org)\nand in the [wiki](http://wiki.libav.org).\n\n### Examples\n\nConding examples are available in the **doc/example** directory.\n\n## License\n\nLibav codebase is mainly LGPL-licensed with optional components licensed under\nGPL. Please refer to the LICENSE file for detailed information.\n’: File name too long

**Comment by wolfbeast** [comment]
Research (Per IRC):
```
[20:36] <@khronosschoty> but when you remove the readme
[20:36] <@khronosschoty> retar the repo the issue goes away
[20:36] <@khronosschoty> or so it seems
[20:37] <@MoonchildPM> It wants to create a symlink to what's in the readme?
[20:37] <@khronosschoty> apparently
[20:37] <@MoonchildPM> Why does it think the readme file is a symlink semaphore?
[20:37] <@khronosschoty> no idea
[20:37] <@khronosschoty> thats the odd thing
[20:37] <@MoonchildPM> either a bug in tar or a bug in the file system...
[20:40] <@MoonchildPM> There's nothing odd in the file. I looked at in in hex view even to make sure and there's no strange control characters or anything.
[20:40] <@khronosschoty> hmm
[20:41] <@MoonchildPM> there's a readme and readme.md of exactly the same length and content there. maybe it trips over that somehow?
[20:41] <@khronosschoty> maybe
```

**Comment by khronosschoty** [comment]
when trying to read the file in vim get permission denied, when trying to use cat to display the README via stdnout you get File name too long. 
`cat  README: File name too long`

>  readlink - print resolved symbolic links or canonical file names (from the readlink man page)

The below out put is what readlink outputs. 
```
schoty@Darkstar:~/notabug.org/SlackBuild-git/network/basilisk/moebius/media/libav$ readlink README
Libav
=====

Libav is a collection of libraries and tools to process multimedia content
such as audio, video, subtitles and related metadata.

## Libraries

* `libavcodec` provides implementation of a wider range of codecs.
* `libavformat` implements streaming protocols, container formats and basic I/O access.
* `libavutil` includes hashers, decompressors and miscellaneous utility functions.
* `libavfilter` provides a mean to alter decoded Audio and Video through chain of filters.
* `libavdevice` provides an abstraction to access capture and playback devices.
* `libavresample` implements audio mixing and resampling routines.
* `libswscale` implements color conversion and scaling routines.

## Tools

* [avconv](http://libav.org/avconv.html) is a command line toolbox to
  manipulate, convert and stream multimedia content.
* [avplay](http://libav.org/avplay.html) is a minimalistic multimedia player.
* [avprobe](http://libav.org/avprobe.html) is a simple analisys tool to inspect
  multimedia content.
* Additional small tools such as `aviocat`, `ismindex` and `qt-faststart`.

## Documentation

The offline documentation is available in the **doc/** directory.

The online documentation is available in the main [website](http://libav.org)
and in the [wiki](http://wiki.libav.org).

### Examples

Conding examples are available in the **doc/example** directory.

## License

Libav codebase is mainly LGPL-licensed with optional components licensed under
GPL. Please refer to the LICENSE file for detailed information.


```

**Comment by wolfbeast** [comment]
Fixed by removing the half-resolved symlink-file in 642d1d4f644852e853782ebbaa7845f528f08e12
